From adff3ad46f9e1330db02d06181defd34b410bbc2 Mon Sep 17 00:00:00 2001
From: John Wright <jsw@debian.org>
Date: Sun, 21 Mar 2010 01:07:25 -0600
Subject: [PATCH] Add makedumpfile

makedumpfile supports a subset of architectures that Linux runs on, as
it needs explicit knowledge of the location and layout of certain kernel
structures on each architecture.
---
 Packages-arch-specific |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index 46495cf..cd12239 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -201,6 +201,7 @@ ltrace: i386 m68k arm armel powerpc s390 sparc ia64 amd64 lpia        # [p] need
 %lustre:  i386 powerpc amd64 ia64				      # ANAIS, no upstream support
 %mac-fdisk: m68k powerpc 					      # Apple Mac fdisk
 %m68kboot: m68k							      # m68k boot loader
+%makedumpfile: amd64 i386 ia64 powerpc				      # not yet ported to other platforms
 mbr: i386 amd64 lpia                                                  # i386 boot loader
 %mcelog: i386 amd64						      # [ANAIS]
 memtest86: i386 amd64 lpia kfreebsd-i386 kfreebsd-amd64 hurd-i386     # i386 memory tester (bypasses OS entirely)
-- 
1.5.6.5

